guide/16-geoenrichment-guides/part4_enrich_collections_and_sedf.ipynb

```diff
@@ -0,0 +1,2937 @@
+{
```

Comment:
<ul><li>typo in 'look'</li><li>Let's loook at data collections for <code>Key Facts, Education and Spending</code>. -&gt; without code styling as these are not actual keys used in the examples below</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>